### DISCUSSION TITLE
Reports not getting generated in Course dashboard

### DISCUSSION THREAD

**Discussion by vj4a** [Sunbird-Lern/Community#191] [Issues]
Hello team,

The status of reports under course dashboard is still shown as processing even after 24 hours. Kindly provide details on which job is to be checked for the same.

![image](https://github.com/Sunbird-Lern/Community/assets/63919015/854044a6-7ae5-426d-9afb-262f3f997cba)

**Comment by AmiableAnil** [comment]
@vj4a Please share the summary of the call here which we had.

cc: @mohitga

**Reply by vj4a** [reply]
@AmiableAnil ,

Forked the https://github.com/Sunbird-Obsrv/sunbird-data-pipeline repo and changed the spark versions to 3.1.3 in below files,

https://github.com/Sunbird-Obsrv/sunbird-data-pipeline/blob/685611b2fdf466926352b4fc351d92a3dd93841e/ansible/roles/analytics-spark-provision/defaults/main.yml#L6

https://github.com/Sunbird-Obsrv/sunbird-data-pipeline/blob/685611b2fdf466926352b4fc351d92a3dd93841e/ansible/roles/data-products-deploy/defaults/main.yml#L49

and then reprovisioned spark.

Used latest tags for Build and deploy for below jobs,
Datapipeline/AnalyticsCore
Datapipeline/Coredataproducts
Lern/Lerndataproducts

Please let know what are the next steps to be taken.
